### ISSUE TITLE
Cannot create drive

### ISSUE THREAD

**Issue by sammacbeth** [andrewosh/hyperdrive-daemon#43]
On linux, I get the following error when trying to create a drive after a fresh install:

```
$ hyperdrive create ~/Hyperdrive/test
✖ Could not create the drive:
TypeError: toMetadata is not a function
```

Some info on my environment:
```
$ hyperdrive status
The Hyperdrive daemon is running:

  API Version:             0
  Daemon Version:          1.9.7
  Client Version:          1.11.1
  Schema Version:          1.8.1
  Hyperdrive Version:      10.8.20
  Fuse Native Version:     2.2.1
  Hyperdrive Fuse Version: 1.2.15

  Holepunchable:           false
  Remote Address:

  Fuse Available:          true
  Fuse Configured:         false

  Uptime:                  0 Days 0 Hours 2 Minutes 11 Seconds

$ node -v
v12.16.2
```

**Comment by andrewosh** [comment]
Thanks Sam -- I'd done some refactoring of the CLI and messed up some of the FUSE commands. Fixed in 1.11.1. Reinstalling `hyperdrive-daemon@beta` should do it.

Also, it looks like you haven't run `hyperdrive fuse-setup` yet -- the status is showing that FUSE is available but isn't configured. This means that your `~/Hyperdrive` directory isn't mounted so the `create` command won't work as expected (it creates a Hyperdrive mounted at some path underneath `~/Hyperdrive`). Sorry for any confusion there! If you run `hyperdrive fuse-setup` before you start the daemon, everything should be mounted correctly the next time you restart.

**Comment by sammacbeth** [comment]
Great that fixed it. Thanks for the quick response and fix!
